data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1179 @@
+			choice
+				`	(Leave.)`
+					goto leave
+				`	(Continue to observe.)`
+			`	The approaching ships continue their descent, paying you no attention. They appear to be ones that were in orbit when you first entered the system. One of them is the same type of model as the one on the ground, likely some type of transport. The other, however, isn't one you've seen before. It looks like some sort of interceptor vessel, though it is definitely larger than something like a Sparrow from human space, maybe even the size of a Headhunter. Perhaps that ship belongs to someone of importance.`
+			`	In any case, considering that the planet is home to more of those monsters from before, and the Vi are here in numbers, it might be best you take your leave now before the situation gets out of control again.`
+			choice
+				`	(Leave.)`
+					goto leave
+				`	(Approach the landing zone.)`
+			`	Throttling your ship forward, you head towards the Vi's landing zone. While the other transport vessel has touched down already, the interceptor is still in the process of landing, but suddenly whips around to face you as you approach. Unlike the initial reaction of the Scin's warships you confronted a few days ago, the Vi are quick to respond to your presence, and in contrast to a few minutes ago, are clearly acknowledging it. A moment later, you receive what sounds like an angry hail from the interceptor.`
+			`	"Tchii ciei escitch tchii iedf teih Voordhte!"`
+			choice
+				`	(Leave now.)`
+				`	(Continue.)`
+					goto continue
+			`	You decide it's best to not enroach on the Vi's area any more, as there's no telling what they'd do if you continued to do so. Once you back off, the interceptor briefly jerks forward at you in a vexed manner, continuing to observe you until you've left the area. While the Scin seemed more open to interaction, the Vi are clearly adversive to it.`
+				decline
+
+			label continue
+			action
+				set "provoked military"
+			`	You decide to continue flying towards the Vi without heeding their warning. With no hesitation, the interceptor begins to charge at you, and the ships on the surface start preparing for takeoff. Even some of the Vi solders on the ground are shouting and gesturing towards your vessel with their weapons. It would seem the Vi are no longer tolerative of your presence.`
+				decline
+
+			label leave
+			action
+			`	You decide to withdraw from the area, and the ships continue on their way towards the surface as if you weren't present. Regardless of whether or not they could have been hostile, getting mixed up with what may be some sort of military exercise could be dangerous, especially with the Vi.`
+				decline
+
+	npc save
+		on kill
+			set "yvx killed"
```

Comment:
same for this variable, maybe set "Imtydei killed"

Reply:
Yvx is the name of the character, who may not always be in their starting ship, so I think it's fine the way it is.